docs/private-dns/connect-devices/mobile-and-desktop/linux.md

```diff
@@ -102,6 +102,21 @@ If you see a notification that you are not connected to AdGuard DNS, most likely
 
 :::
 
+## Use EDNS (Extended DNS)
+
+EDNS extends the DNS protocol, enabling larger UDP packets to carry additional data. In AdGuard DNS, it allows passing the DeviceID in plain DNS using an extra parameter.
+
+DeviceID, an eight-digit hexadecimal identifier (e.g., `1a2b3c4d`), helps link DNS requests to specific devices. For encrypted DNS, this ID is part of the domain (e.g., `1a2b3c4d.d.adguard-dns.com`). For unencrypted DNS, EDNS is required to localize this identifier.
```

Comment:
I'm not sure what the last sentence is supposed to convey.  Perhaps, something closer to “… to transfer this identifier”?

Reply:
same comment. wrong word choice — "localize" can't be used here